fundingmanager.go

```diff
@@ -43,6 +41,13 @@ const (
 	// TODO(roasbeef): add command line param to modify
 	maxFundingAmount = btcutil.Amount(1 << 24)
 
+	// minRemoteDelay and maxRemoteDelay is the extremes of the CSV delay
+	// we will require the remote to use for its commitment transaction.
+	// The actual delay we will require will be somewhere between these
+	// values, depending on channel size.
+	minRemoteDelay = 4
+	maxRemoteDelay = 144
```

Comment:
A max delay of 144 blocks (1 day) is insufficient. Keep in mind that this parameters serves as the minimum time delta a node can go w/o coming online (assuming no watch towers). To lie on the conservative side of things, I'd say the max should be somewhere around 3024 blocks (1.5 retarget periods). In order to ensure a channel of like 10k satoshis doesn't trigger such a high CSV value, we can use a piecewise function to place channel amounts into various buckets. 

Similarly, a min delay of 4 is also too low IMO. This means that there's a 40 minute window of vulnerability. This one is a bit more subjective as there may be situations where you're OK with a small CSV value (possibly making a channel with a friend, or frequent business partner). With that said, what do you think of a min value of 72 (12 hours) or 144 (24 hours)? 

I haven't finished going over the remainder of the commits, but we should also be using these values as bounds for rejecting when the remote party proposes a CSV value.

Reply:
Totally agree that a minimum of `4` is too low. I set it that low because `4` was the previous default. 

I think a minimum of `144` is fine, and I don't think it is a problem to wait `3024` if ~$3 is at stake, so `3024` sounds good.

Reply:
However, maybe 3 weeks is a bit low if unreliable peers require you to use that. Also autopilot will open max value channels, which can tie up a lot of your funds for weeks. `2016` is maybe a better number.

Reply:
Made min `144`, max `2016`.